frontend/src/employee-frontend/components/child-information/Assistance.tsx

```diff
@@ -41,6 +43,12 @@ export default React.memo(function Assistance({ id, startOpen }: Props) {
         {permittedActions.has('READ_ASSISTANCE_ACTION') && (
           <AssistanceAction id={id} />
         )}
+        {permittedActions.has('READ_ASSISTANCE_NEED_DECISIONS') && (
```

Comment:
Should also be hidden behind the `specialNeedsDecisions` feature flag (which should probably be renamed to `assistanceNeedDecisions`)